cla_public/templates/base.html

```diff
@@ -54,38 +54,31 @@
       {% endfor %}
     {% endif %}
   {% endif %}
-  <div class="laa-flee-button-container
+  <main class="govuk-main-wrapper " id="main-content" role="main" lang="{{ request.cookies.get('locale', 'en')[:2] }}">
     {% if is_quick_exit_enabled(session) %}
-      laa-flee-button-enabled
+      <div class="laa-flee-button-container laa-flee-button-enabled">
```

Comment:
Does this div need to exist? The `laa-flee-button-enabled` class has been removed and the styling for the element is handled by GDS. If the `laa-flee-button-container` class should be present on the button then the class could be moved to the child div.